spec/docs/termmap.md

```diff
@@ -180,14 +171,6 @@ before they can be used in IRIs.
 [=Template-valued term maps=] are a convenient way of percent-encoding data values.
 </aside>
 
-<aside class="note">
-[=Constant-valued term maps=] are not considered as having a term type,
-and specifying `rml:termType` on these [=term maps=] has no effect.
-The type of the [=generated RDF term=] is determined directly by the value of `rml:constant`:
-If it is an [IRI](), then an [IRI]() will be generated;
-if it is a literal, a literal will be generated.
-</aside>
```

Comment:
Why has this been removed? This is an important detail. This clarifies that we cannot coerce constants. And that seems quite logical to me.